fe/src/main/java/org/apache/doris/analysis/CreateMaterializedViewStmt.java

```diff
@@ -167,15 +167,25 @@ private void analyzeSelectClause() throws AnalysisException {
             } else if (selectListItem.getExpr() instanceof FunctionCallExpr) {
                 FunctionCallExpr functionCallExpr = (FunctionCallExpr) selectListItem.getExpr();
                 String functionName = functionCallExpr.getFnName().getFunction();
+                Expr defineExpr = null;
                 // TODO(ml): support REPLACE, REPLACE_IF_NOT_NULL only for aggregate table, HLL_UNION, BITMAP_UNION
                 if (!functionName.equalsIgnoreCase("sum")
                         && !functionName.equalsIgnoreCase("min")
-                        && !functionName.equalsIgnoreCase("max")) {
+                        && !functionName.equalsIgnoreCase("max")
+                        && !functionName.equalsIgnoreCase("bitmap_union")
+                        && !functionName.equalsIgnoreCase("hll_union")) {
                     throw new AnalysisException("The materialized view only support the sum, min and max aggregate "
                                                         + "function. Error function: " + functionCallExpr.toSqlImpl());
                 }
                 Preconditions.checkState(functionCallExpr.getChildren().size() == 1);
                 Expr functionChild0 = functionCallExpr.getChild(0);
+
+                if (functionName.equalsIgnoreCase("bitmap_union") || functionName.equalsIgnoreCase("hll_union")) {
```

Comment:
I think we need an extensible framework to handle the expression part of the materialized view more elegantly, rather than simply judging a few functions.
This can prepare for more complex expressions later.

Reply:
I agree with you. But schema change and  expression compute  logical is non-universal. so we can only support few function now.